### ISSUE TITLE
Field is effectively useless - "number of liquidated contracts" in Depth websocket message

### ISSUE THREAD

**Issue by linmaosong2018** [okcoin-okex/API-docs-OKEx.com#212]
According to document here: https://www.okex.com/docs/en/#futures_ws-depth

<img width="646" alt="image" src="https://user-images.githubusercontent.com/39483900/53170307-b9071280-35d7-11e9-8dc2-a4fe0a4b04d5.png">

As explained by the highlighted part, the 3rd field is the number of liquidated contracts (presumably the given price is their entry price). 

In reality, in the depth update messages, this field always seem to have zero value. Furthermore, the price level of the liquidated contracts doesn't seem to be among depth update messages at all.

Here is a demonstration. 

Test setup
1. Establish a websocket connection to receive ```ETH-USD-190329``` depth data
2. Keep refreshing liquidation list of ```ETH-USD-190329``` through the REST endpoint ```GET /api/futures/v3/instruments/ETH-USD-190329/liquidation```

Observations
1. At 2019-02-21T11:23:37Z, the REST endpoint returns a new record:
```
[{"loss":0.27768301,"size":455,"price":142.619,"created_at":"2019-02-21T11:23:26.000Z","type":3,"instrument_id":"ETH-USD-190329"}  <...omitted>
```

2. As per the ```created_at``` field, this liquidation record was created at ```11:23:26```

3. Now go back to check all the messages through the websocket between 11:23:25 and 11:23:27
```
2019-02-21T11:23:25Z {"table":"futures/depth","action":"update","data":[{"instrument_id":"ETH-USD-190329","asks":[[143.267,3,0,1],[143.268,0,0,0],[143.27,25,0,1],[143.28,71,0,2],[143.288,25,0,2],[143.297,1011,0,3],[143.314,0,0,0],[143.318,0,0,0],[143.327,48,0,2],[143.328,57,0,1],[143.331,69,0,2],[143.332,18,0,1],[143.335,160,0,2],[143.34,250,0,1],[143.342,56,0,1],[143.343,88,0,1],[143.346,0,0,0],[143.348,0,0,0],[143.349,80,0,1],[143.35,400,0,1],[143.36,80,0,1],[143.361,80,0,0],[143.369,0,0,0],[143.378,75,0,1],[143.38,29,0,1],[143.384,108,0,2],[143.386,80,0,0],[143.388,0,0,0],[143.389,87,0,1],[143.393,558,0,1],[143.398,45,0,1],[143.403,87,0,2],[143.409,144,0,2],[143.41,75,0,1],[143.415,0,0,0],[143.417,119,0,2],[143.419,87,0,1],[143.424,85,0,1],[143.425,89,0,3],[143.426,0,0,0],[143.43,0,0,0],[143.436,76,0,1],[143.444,123,0,2],[143.447,79,0,1],[143.449,148,0,2],[143.45,0,0,0],[143.451,0,0,0],[143.452,201,0,1],[143.46,0,0,0],[143.463,0,0,0],[143.466,114,0,1],[143.478,87,0,2],[143.491,79,0,1],[143.498,0,0,0],[143.505,430,0,1],[143.506,11,0,1],[143.539,135,0,2],[143.54,11,0,1],[143.541,0,0,0],[143.548,0,0,0],[143.551,76,0,1],[143.555,350,0,1],[143.556,0,0,0],[143.565,293,0,1],[143.571,0,0,0],[143.574,239,0,1],[143.577,53,0,2],[143.592,0,0,0],[143.594,26,0,1],[143.608,293,0,1],[143.611,17,0,2],[143.612,5,0,1],[143.619,411,0,2],[143.626,114,0,1],[143.638,76,0,1],[143.641,53,0,2],[143.652,826,0,3],[143.656,0,0,0],[143.657,0,0,0],[143.658,33,0,1],[143.666,11,0,1],[143.679,230,0,1],[143.693,291,0,2],[143.717,11,0,1],[143.737,0,0,0],[143.751,26,0,1],[143.752,0,0,0],[143.763,400,0,1],[143.772,11,0,1],[143.792,30,0,2],[143.809,356,0,3],[143.815,26,0,1],[143.837,0,0,0],[143.866,340,0,1],[143.884,26,0,1],[143.913,440,0,2],[143.939,0,0,0],[143.942,0,0,0],[143.943,0,0,0],[143.95,0,0,0],[143.955,0,0,0],[143.968,0,0,0],[143.97,0,0,0],[143.976,0,0,0],[143.989,0,0,0],[143.997,0,0,0],[143.999,0,0,0],[144.0,0,0,0],[144.012,0,0,0],[144.047,0,0,0],[144.049,0,0,0],[144.05,0,0,0],[144.075,0,0,0],[144.08,0,0,0],[144.087,0,0,0],[144.09,0,0,0],[144.095,0,0,0],[144.1,0,0,0],[144.104,0,0,0]],"bids":[[143.25,0,0,0],[143.245,0,0,0],[143.234,0,0,0],[143.21,3648,0,3],[143.17,0,0,0],[143.148,0,0,0],[143.146,2,0,1],[143.144,0,0,0],[143.136,400,0,1],[143.126,11,0,1],[143.113,1,0,1],[143.111,0,0,0],[143.107,11,0,1],[143.102,50,0,1],[143.099,0,0,0],[143.096,0,0,0],[143.092,350,0,1],[143.087,7245,0,20],[143.085,100,0,2],[143.084,1,0,1],[143.082,0,0,0],[143.077,50,0,1],[143.069,0,0,0],[143.055,695,0,1],[143.043,21,0,1],[143.021,250,0,1],[143.015,406,0,2],[142.986,0,0,0],[142.937,0,0,0],[142.936,6,0,1],[142.935,0,0,0],[142.91,236,0,4],[142.802,0,0,0],[142.758,0,0,0],[142.757,12,0,1],[142.756,0,0,0],[142.731,72,0,3],[142.402,0,0,0],[142.401,0,0,0],[142.4,266,0,3],[142.377,38,0,2],[142.372,40,0,1],[142.12,88,0,1],[142.115,1,0,1],[142.11,142,0,1],[142.1,1164,0,9],[142.089,200,0,1],[142.081,300,0,1],[142.076,728,0,1]],"timestamp":"2019-02-21T11:23:25.471Z","checksum":1422398755}]}
```

```
2019-02-21T11:23:26Z {"table":"futures/depth","action":"update","data":[{"instrument_id":"ETH-USD-190329","asks":[[143.267,0,0,0],[143.269,28,0,2],[143.288,0,0,0],[143.289,4,0,1],[143.297,761,0,2],[143.303,50,0,1],[143.31,109,0,2],[143.314,50,0,1],[143.317,290,0,4],[143.325,75,0,1],[143.326,61,0,1],[143.33,61,0,1],[143.331,50,0,1],[143.332,80,0,1],[143.334,69,0,1],[143.335,0,0,0],[143.338,0,0,0],[143.34,0,0,0],[143.343,0,0,0],[143.345,461,0,2],[143.349,0,0,0],[143.35,42,0,1],[143.351,37,0,1],[143.356,88,0,1],[143.361,0,0,0],[143.363,0,0,0],[143.364,0,0,0],[143.37,0,0,0],[143.373,0,0,0],[143.375,42,0,1],[143.383,71,0,1],[143.384,0,0,0],[143.385,80,0,1],[143.386,0,0,0],[143.389,0,0,0],[143.39,0,0,0],[143.395,0,0,0],[143.397,54,0,1],[143.406,0,0,0],[143.409,29,0,1],[143.415,146,0,2],[143.417,42,0,1],[143.419,0,0,0],[143.424,0,0,0],[143.425,0,0,0],[143.43,54,0,1],[143.438,0,0,0],[143.44,0,0,0],[143.443,123,0,2],[143.444,0,0,0],[143.449,5,0,1],[143.452,0,0,0],[143.457,21,0,1],[143.458,33,0,1],[143.478,185,0,3],[143.479,0,0,0],[143.482,64,0,1],[143.485,20,0,2],[143.486,33,0,1],[143.511,0,0,0],[143.515,33,0,1],[143.518,250,0,0],[143.52,193,0,3],[143.539,114,0,1],[143.55,493,0,2],[143.551,174,0,2],[143.555,64,0,1],[143.557,33,0,1],[143.561,293,0,1],[143.564,392,0,2],[143.565,0,0,0],[143.566,0,0,0],[143.569,22,0,2],[143.57,162,0,4],[143.573,0,0,0],[143.574,0,0,0],[143.588,0,0,0],[143.599,133,0,2],[143.605,563,0,2],[143.608,0,0,0],[143.61,0,0,0],[143.611,10,0,1],[143.612,5,0,0],[143.637,436,0,3],[143.641,117,0,3],[143.646,33,0,1],[143.648,465,0,1],[143.649,0,0,0],[143.652,340,0,1],[143.674,0,0,0],[143.697,34,0,2],[143.699,0,0,0],[143.749,26,0,1],[143.75,140,0,3],[143.752,33,0,1],[143.791,0,0,0],[143.792,30,0,1],[143.805,600,0,2],[143.824,0,0,0],[143.828,123,0,1],[143.829,0,0,0],[143.926,300,0,1],[143.939,200,0,1],[143.942,10,0,1],[143.943,395,0,1],[143.95,2,0,1],[143.955,4,0,1],[143.968,200,0,1],[143.97,3137,0,2],[143.976,2937,0,1],[143.989,2018,0,3],[143.997,1,0,1],[143.999,300,0,1],[144.0,11248,0,5],[144.009,1800,0,1]],"bids":[[143.251,323,0,6],[143.25,5,0,1],[143.235,154,0,1],[143.211,2019,0,3],[143.21,3647,0,3],[143.2,3756,0,11],[143.19,17,0,1],[143.147,189,0,1],[143.126,0,0,0],[143.117,50,0,1],[143.113,0,0,0],[143.107,0,0,0],[143.102,0,0,0],[143.095,250,0,1],[143.092,0,0,0],[143.084,0,0,0],[143.078,613,0,3],[143.077,0,0,0],[143.071,0,0,0],[143.066,283,0,2],[143.055,0,0,0],[143.051,250,0,1],[143.043,0,0,0],[143.041,700,0,1],[143.021,0,0,0],[143.02,7172,0,2],[143.015,1,0,1],[143.01,1573,0,4],[142.983,0,0,0],[142.941,300,0,1],[142.936,0,0,0],[142.911,25,0,1],[142.91,41,0,4],[142.909,16,0,2],[142.908,250,0,1],[142.897,0,0,0],[142.891,6,0,1],[142.855,200,0,1],[142.84,200,0,1],[142.812,0,0,0],[142.798,196,0,1],[142.757,0,0,0],[142.733,0,0,0],[142.731,82,0,4],[142.729,200,0,1],[142.712,12,0,1],[142.565,942,0,2],[142.534,1035,0,1],[142.4,242,0,2],[142.377,2,0,1],[142.375,32,0,2],[142.373,20,0,1],[142.1,0,0,0],[142.089,0,0,0],[142.081,0,0,0],[142.076,0,0,0]],"timestamp":"2019-02-21T11:23:26.070Z","checksum":-149588399}]}
```

```
2019-02-21T11:23:26Z {"table":"futures/depth","action":"update","data":[{"instrument_id":"ETH-USD-190329","asks":[[143.233,7264,0,1],[143.251,5381,0,2],[143.269,120,0,4],[143.279,26,0,2],[143.294,0,0,0],[143.303,0,0,0],[143.31,34,0,1],[143.311,36,0,2],[143.314,0,0,0],[143.317,61,0,1],[143.321,250,0,1],[143.325,75,0,0],[143.328,0,0,0],[143.329,57,0,1],[143.331,0,0,0],[143.332,0,0,0],[143.334,0,0,0],[143.342,0,0,0],[143.343,55,0,1],[143.354,50,0,1],[143.356,250,0,1],[143.36,0,0,0],[143.361,80,0,1],[143.368,6,0,2],[143.378,0,0,0],[143.38,0,0,0],[143.383,0,0,0],[143.385,0,0,0],[143.386,167,0,2],[143.388,88,0,1],[143.397,58,0,3],[143.398,0,0,0],[143.403,10,0,1],[143.404,77,0,1],[143.409,0,0,0],[143.41,0,0,0],[143.411,28,0,3],[143.415,0,0,0],[143.416,233,0,3],[143.418,119,0,2],[143.421,0,0,0],[143.423,85,0,1],[143.424,85,0,1],[143.436,0,0,0],[143.437,148,0,2],[143.447,0,0,0],[143.449,17,0,2],[143.45,74,0,2],[143.456,75,0,1],[143.458,0,0,0],[143.459,0,0,0],[143.469,60,0,1],[143.47,1,0,1],[143.476,321,0,2],[143.478,98,0,1],[143.479,72,0,1],[143.485,41,0,2],[143.486,0,0,0],[143.491,0,0,0],[143.499,0,0,0],[143.506,0,0,0],[143.512,250,0,1],[143.518,0,0,0],[143.519,81,0,2],[143.52,100,0,1],[143.54,0,0,0],[143.55,143,0,1],[143.551,98,0,1],[143.556,21,0,1],[143.557,0,0,0],[143.561,0,0,0],[143.564,153,0,1],[143.57,178,0,5],[143.571,5,0,1],[143.572,400,0,4],[143.577,392,0,2],[143.591,293,0,1],[143.595,239,0,1],[143.598,21,0,1],[143.599,100,0,1],[143.605,270,0,1],[143.612,0,0,0],[143.619,400,0,1],[143.634,493,0,3],[143.637,251,0,2],[143.638,0,0,0],[143.645,21,0,1],[143.646,0,0,0],[143.648,0,0,0],[143.658,172,0,1],[143.666,0,0,0],[143.667,530,0,2],[143.696,21,0,1],[143.697,1,0,1],[143.712,0,0,0],[143.715,720,0,3],[143.717,0,0,0],[143.748,18,0,1],[143.75,172,0,4],[143.751,57,0,3],[143.752,0,0,0],[143.772,0,0,0],[143.792,220,0,2],[143.805,200,0,1],[143.846,400,0,1],[143.88,340,0,1],[143.895,941,0,1],[144.012,200,0,1],[144.047,4,0,1],[144.049,4,0,1],[144.05,500,0,1]],"bids":[[143.251,0,0,0],[143.25,0,0,0],[143.235,0,0,0],[143.211,1971,0,6],[143.168,50,0,1],[143.16,469,0,3],[143.156,200,0,1],[143.127,728,0,3],[143.108,11,0,1],[143.099,1,0,1],[143.096,250,0,1],[143.095,0,0,0],[143.085,0,0,0],[143.082,250,0,1],[143.081,2,0,2],[143.078,0,0,0],[143.067,521,0,1],[143.066,33,0,1],[143.061,700,0,1],[143.052,250,0,1],[143.051,99,0,1],[143.041,0,0,0],[143.018,250,0,1],[143.01,473,0,3],[142.992,24,0,2],[142.909,256,0,3],[142.908,0,0,0],[142.907,200,0,1],[142.892,9,0,1],[142.89,10,0,1],[142.812,200,0,1],[142.73,200,0,1],[142.729,5,0,1],[142.714,18,0,1],[142.711,20,0,1],[142.375,84,0,3],[142.373,84,0,2],[142.372,0,0,0],[142.358,36,0,1],[142.17,0,0,0],[142.16,0,0,0],[142.158,0,0,0],[142.133,0,0,0],[142.127,0,0,0],[142.123,0,0,0],[142.12,0,0,0],[142.115,0,0,0],[142.11,0,0,0]],"timestamp":"2019-02-21T11:23:26.639Z","checksum":-1120956897}]}
```

```
2019-02-21T11:23:27Z {"table":"futures/depth","action":"update","data":[{"instrument_id":"ETH-USD-190329","asks":[[143.233,6557,0,1],[143.25,1,0,1],[143.269,19,0,1],[143.279,0,0,0],[143.28,132,0,3],[143.284,50,0,1],[143.286,75,0,1],[143.291,201,0,1],[143.294,130,0,2],[143.297,921,0,4],[143.301,75,0,1],[143.303,61,0,1],[143.307,400,0,1],[143.311,69,0,1],[143.312,80,0,1],[143.314,62,0,1],[143.317,0,0,0],[143.321,0,0,0],[143.322,61,0,1],[143.323,80,0,2],[143.324,0,0,0],[143.325,0,0,0],[143.326,0,0,0],[143.327,90,0,3],[143.329,57,0,0],[143.33,0,0,0],[143.332,2,0,1],[143.343,55,0,0],[143.345,0,0,0],[143.35,0,0,0],[143.352,42,0,1],[143.356,0,0,0],[143.36,54,0,1],[143.361,0,0,0],[143.368,0,0,0],[143.374,0,0,0],[143.375,0,0,0],[143.376,88,0,1],[143.379,75,0,1],[143.38,29,0,1],[143.386,87,0,1],[143.388,0,0,0],[143.389,180,0,2],[143.39,85,0,1],[143.394,54,0,1],[143.397,0,0,0],[143.4,43,0,1],[143.402,33,0,1],[143.409,152,0,3],[143.41,77,0,1],[143.411,26,0,1],[143.412,11,0,1],[143.418,75,0,1],[143.423,44,0,1],[143.424,0,0,0],[143.425,122,0,1],[143.43,0,0,0],[143.44,11,0,1],[143.443,0,0,0],[143.45,0,0,0],[143.466,168,0,2],[143.468,272,0,1],[143.474,11,0,1],[143.475,33,0,1],[143.476,120,0,1],[143.498,122,0,1],[143.511,11,0,1],[143.516,0,0,0],[143.539,168,0,2],[143.54,350,0,1],[143.552,200,0,2],[143.553,11,0,1],[143.554,521,0,5],[143.559,0,0,0],[143.56,239,0,1],[143.561,33,0,1],[143.57,55,0,4],[143.577,42,0,1],[143.591,0,0,0],[143.594,0,0,0],[143.595,0,0,0],[143.598,314,0,2],[143.6,211,0,3],[143.626,168,0,2],[143.632,0,0,0],[143.634,678,0,3],[143.641,91,0,2],[143.643,0,0,0],[143.651,11,0,1],[143.667,65,0,1],[143.693,265,0,1],[143.706,11,0,1],[143.718,0,0,0],[143.729,0,0,0],[143.751,31,0,2],[143.815,0,0,0],[143.817,2876,0,1],[143.828,0,0,0],[143.884,0,0,0],[143.999,0,0,0],[144.0,0,0,0],[144.009,0,0,0],[144.012,0,0,0],[144.047,0,0,0],[144.049,0,0,0],[144.05,0,0,0]],"bids":[[143.232,136,0,1],[143.211,1966,0,6],[143.19,0,0,0],[143.177,202,0,2],[143.173,144,0,1],[143.168,0,0,0],[143.161,0,0,0],[143.16,400,0,1],[143.159,0,0,0],[143.147,0,0,0],[143.127,717,0,2],[143.117,0,0,0],[143.108,130,0,1],[143.099,0,0,0],[143.096,0,0,0],[143.089,2183,0,5],[143.082,0,0,0],[143.081,0,0,0],[143.077,100,0,1],[143.076,1,0,1],[143.074,0,0,0],[143.067,0,0,0],[143.061,0,0,0],[143.054,50,0,1],[143.052,0,0,0],[143.018,0,0,0],[143.008,1128,0,2],[143.0,130827,0,56],[142.975,1,0,1],[142.91,29,0,2],[142.909,6,0,2],[142.907,0,0,0],[142.893,210,0,3],[142.891,0,0,0],[142.874,13,0,2],[142.873,5,0,1],[142.812,0,0,0],[142.731,58,0,2],[142.73,0,0,0],[142.712,0,0,0],[142.695,26,0,2],[142.391,2060,0,1],[142.375,56,0,2],[142.373,64,0,1],[142.17,1,0,1],[142.16,1,0,1],[142.158,176,0,1],[142.133,2000,0,1],[142.127,2,0,1],[142.123,287,0,2],[142.12,88,0,1],[142.115,1,0,1],[142.11,142,0,1],[142.1,1164,0,9]],"timestamp":"2019-02-21T11:23:27.256Z","checksum":948526202}]}
```

The websocket delivered 4 messages in total. None of the messages contained any liquidation record (the 3rd field always had zero value), none of them contained the liquidation price level ```142.619```

In conclusion, this field is effectively useless.

**Comment by linmaosong2018** [comment]
Also, please don't brush off this report with "you just need to follow our document", which seems to have been the default standard answer to issues raised here.

**Comment by Swiftnesses** [comment]
@linmaosong2018 Did you ever solve this?

Further, I'm not able to get a valid response on the rest endpoint when I include 'liquidation', do you have a working example?

**Comment by linmaosong2018** [comment]
@Swiftnesses 
* Websocket issue - no, still unsolved. No update/reply from okex
* Rest endpoint - yes it appears to work for me ```curl "https://www.okex.com/api/futures/v3/instruments/BTC-USD-190628/liquidation?status=1"```

**Comment by Swiftnesses** [comment]
Hi @linmaosong2018 thanks for the quick response, that works for me...

I'm trying to replicate https://twitter.com/whalecalls?lang=en, I assume they get the info via the rest API and not via websockets - I don't see a way to see specific order liquidations via websockets?

Any help would be greatly appreciated.
